mlflow/server/js/src/experiment-tracking/components/runs-charts/hooks/useSampledMetricHistory.tsx

```diff
@@ -110,7 +110,7 @@ export const useSampledMetricHistory = (params: {
 
     // Base fetching function, used for both initial call and subsequent auto-refresh calls
     const fetchMetricsFn = async (isAutoRefreshing = false) => {
-      const runUuids = runUuidsSerialized.split(',').filter((runUuid: string) => runUuid !== '');
+      const runUuids = runUuidsSerialized.split(',');
```

Comment:
Can we leave this line with `.filter`? this is from reverted code, but is actually a useful fix. I can help with backsync conflicts if any